handler/handler_test.go

```diff
@@ -59,8 +62,14 @@ func (m *MockMetricStore) Ready() error {
 	return nil
 }
 
+func newMockMetricStore() MockMetricStore {
```

Comment:
Not a big deal, but in this scenario where the usage of the MockMetricStore is purely internal, and all what this constructor does is to create an empty map, I'd prefer to keep things simple and not have a constructor function.

Reply:
:+1: